lib/active_fedora/aggregation/base_extension.rb

```diff
@@ -15,8 +15,17 @@ def aggregated_by
       proxy_class.where(proxyFor_ssim: id).map(&:container)
     end
 
+    def ordered_by
+      ordered_by_ids.map{ |x| ActiveFedora::Base.find(x) }
```

Comment:
This could result in lots of hits to Fedora. Should we be worried about performance here? I'm not sure how this is used, so I ask in some ignorance whether this might be lazily iterated. If not a good fit, OK.